pkg/job/job.go

```diff
@@ -19,7 +19,7 @@ func (s Status) Pending() bool {
 	return true
 }
 
-func (s Status) Completed() bool {
+func (s Status) JobComplete() bool {
```

Comment:
job is not necessary i think method's names should be completed and failed since the package and type `JobStatus` says what this adjective is about, but ofc if you would like to have `Job` in the method name, pls change also `Pending` method

Reply:
I cannot create method `Failed` since Status which is in fact batch.JobStatus already has this field.

I will change Pending to JobPending.